cas-server-core-util/src/main/java/org/apereo/cas/util/AbstractJacksonBackedStringSerializer.java

```diff
@@ -21,13 +23,14 @@
 import java.io.Reader;
 import java.io.StringWriter;
 import java.io.Writer;
+import java.util.stream.Collectors;
 
 /**
  * Generic class to serialize objects to/from JSON based on jackson.
  * @author Misagh Moayyed
  * @since 4.1
  */
-public abstract class AbstractJacksonBackedJsonSerializer<T> implements JsonSerializer<T> {
+public abstract class AbstractJacksonBackedStringSerializer<T> implements StringSerializer<T> {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Type Javadoc comment is missing an @param <T> tag.](https://www.codacy.com/app/mmoayyed/cas/file/2643620578/issues/source?bid=3256115&fileBranchId=3322154#l33)